scripts/callbacks_generator/gen_callbacks.py

```diff
@@ -841,7 +874,12 @@ def generate_from_collector(api: CallbackDataCollector, output):
     cg.generate_full_source(api)
 
   @staticmethod
-  def generate_asserts_from_collector(api: CallbackDataCollector, output):
+  def generate_asserts_from_collector(api: CallbackDataCollector, output, args):
+    entrypoints = os.path.join(os.path.dirname(args.config[0]), 'entrypoints.h')
```

Comment:
Wouldn't it make sense to generate the full source code (`generate_from_collector`) from the `entrypoints.h` file as well, so that the output (`generated_callbacks.cc`) contains only APIs supported by tizen_interop and you don't need to include extra headers at all?

Reply:
Processing entrypoint.h is troublesome - would need to recursively look for files taking into account the configures include dirs. I assumed that patterns in ffigen config (include-directives) will be mostly in sync with entrypoints.h. But I will try doing that. First experiment with loading only files listed in entrypoints.h (without following include directives) shows a significant difference in the number of callbacks found.

Reply:
If that's too much work, let's just keep it the way it is.